client/my-sites/plan-features/bottom.jsx

```diff
@@ -0,0 +1,27 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+export default class PlanFeaturesBottom extends React.PureComponent {
+	getContainer() {
+		if ( ! this.container || ! this.container.offsetWidth ) {
+			this.container = document.querySelector( '.plans-features-main__bottom' );
+		}
+
+		return this.container;
+	}
+
+	render() {
+		const { children } = this.props;
+		const container = this.getContainer();
+
+		if ( ! children || ! container ) {
+			return null;
+		}
+
+		return ReactDOM.createPortal( children, container );
```

Comment:
What's the reason for using a portal here? Doesn't that make things more complex and less easily predictable then simply using the existing component hierarchy?